config/webpack.mobile.target.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const path = require('path')
+const webpack = require('webpack')
+
+const production = /:production$/.test(process.env.NODE_ENV)
+
+module.exports = {
+  entry: path.resolve(__dirname, '../mobile/src/main'),
+  output: {
+    path: path.resolve(__dirname, '../mobile/www')
+  },
+  module: {
+    noParse: [
+      /localforage\/dist/
+    ]
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      __ALLOW_HTTP__: !production,
+      __TARGET__: JSON.stringify('mobile')
```

Comment:
The `stringify` is mandatory in `webpack.definePlugin` config (https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin). So the condition to test is just `if (__TARGET__ === 'mobile') { }`